server/src/test/java/org/elasticsearch/index/mapper/BooleanScriptFieldTypeTests.java

```diff
@@ -308,7 +308,7 @@ public void testDualingQueries() throws IOException {
             String source = "{\"foo\": " + values + "}";
             XContentParser parser = createParser(JsonXContent.jsonXContent, source);
             SourceToParse sourceToParse = new SourceToParse("test", "test", new BytesArray(source), XContentType.JSON);
-            DocumentParserContext ctx = new TestDocumentParserContext(null, null, null, null, sourceToParse) {
+            DocumentParserContext ctx = new TestDocumentParserContext(MappingLookup.EMPTY, null, null, null, sourceToParse) {
```

Comment:
There is another 0-argument ctor in TestDocumentParserContext that seems unused, maybe we should delete it or otherwise also use MappingLookup.EMPTY as first argument there to avoid suprises later.

Reply:
Good catch - that's actually causing some percolator failures, will switch it to always use MappingLookup.EMPTY